.github/workflows/push.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
           path: self
 
       - name: Validate chart
-        uses: actions/github-script@v6
+        uses: actions/github-script@v6.4.1
```

Comment:
any reason we're pinning to this specific version?

Reply:
the reason I ask is because by using `v6` we always use the latest v6 version, so I don't think this is required for this particular fix, and it will require us to regularly update when new minor or patch versions are released